README.md

```diff
@@ -0,0 +1,187 @@
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 CAPSTONE PROYECT <a name="about-project"></a>
+
+**[CAPSTONE PROYECT Urban Art Festival Creativity in the Streets 2023]** is a proyect was created to exercise good code and version control practices in the area of ​​web development.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>VSC</li>
+    <li>Node.js</li>
+    <li>Flexblox</li>
+    <li>Figma</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+-**Learn to use new tools for the frontend**
+-**Learn to make a professional webpage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://fernando426.github.io/Portfolio-mobile-website2/)
```

Comment:
- [ ]please kindly deploy this project and add the correct live link like the previous reviewer noted. currently it has a link to your portfolio